spec/unit/ace/transport_app_spec.rb

```diff
@@ -25,18 +25,13 @@ def app
   let(:file_cache) { instance_double(BoltServer::FileCache, 'file_cache') }
   let(:task_response) { instance_double(Bolt::ResultSet, 'task_response') }
   let(:plugins) { instance_double(ACE::PluginCache, 'plugin_cache') }
-  let(:response) { instance_double(Bolt::Result, 'response') }
```

Comment:
General improvement to these tests is to use a real instance of a result instead of an instance double :smiley: